src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,36 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health) {
+    this.name = name;
+    this.health = health || 100;
+    Animal.alive.push(this);
+  };
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health) {
+    super(name, health);
+    this.hidden = false;
+  }
+
+  hide() {
+    this.hidden = true;
+  }
```

Comment:
```suggestion
  hide() {
    this.hidden = !this.hidden;
  }
```
<img width="863" alt="image" src="https://user-images.githubusercontent.com/38065883/173184414-3449c2e9-ec37-4fc5-b0a3-f38e89cf317f.png">